packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -1042,6 +1074,46 @@ export class AttributesController extends ConsumptionBaseController {
         }
     }
 
+    private async handleDefault(attribute: LocalAttribute): Promise<void> {
```

Comment:
If find the method's name a little vague ("handle" doesn't suggest to me what's really going on here). How about something like `setAlternativeDefault`?

Reply:
What about `overwriteDefault`?

Reply:
This poses the same problem: The default is only overwritten, in case the Attribute I delete was the former default. So, if in the function `executeFullAttributeDeletionProcess` a function called `overwriteDefault` is called, this would cause confusion. In my opinion, `transmitDefault` has the advantage that you can figure, that something can only be transmitted, if it was there before.